aips/resource_groups.md

```diff
@@ -75,31 +73,33 @@ The motivation of using a `struct` is that
 2. It can build upon the existing storage model that knows how to read and write data stored at  `StructTag`s. Thus it limits the implementation impact to the VM and readers of storage, storage can remain agnostic to this change.
 3. Only `struct` and `fun` can have attributes, which in turn let’s us define additional parameters like `scope`.
 
-Each entry in a resource group is identified by the `resource_group` attribute:
+Each entry in a resource group is identified by the `resource_group_member` attribute:
 
 ```move
-#[resource_group(container = aptos_framework::object::ObjectGroup)]
+#[resource_group_member(group = aptos_framework::object::ObjectGroup)]
 struct Object has key {
```

Comment:
If this `resource_group_member` attribute is present, does it mean we'll enforce that these structs can only be used as part of that resource group, and not in any other resource group / on its own?

Reply:
yes, will make sure that is clear in the writing.